code/game/objects/structures/window.dm

```diff
@@ -523,7 +523,8 @@
 	icon_state = "light0"
 	anchored = TRUE
 	desc = "A remote control switch for polarized windows."
-	var/range = 7
+	var/area/button_area
+	var/range = 0
```

Comment:
Auto-doc these, please.

Also, I think you might want to make the "room-mode" marked by a define, and maybe set it to -1? I think it's a little confusing as it is rn